### PR TITLE
TabBarIOS tintColor

### PR DESCRIPTION
**Pull request by tsunammis**
[Origin Pull request](https://github.com/facebook/react-native/pull/961) from [cmcewen](https://github.com/cmcewen)

All the work have been done by @cmcewen, I just rebased his work with the master.

### CODE REVIEW

**Comment by ericvicenti** [comment]
Lucky number 1337!

Can you expand upon the tab bar example code to show this new functionality?

**Comment by tsunammis** [comment]
@ericvicenti do you know how launch UIExplorer e2e test directly on my desktop ?

I tried some commands

```bash
cd Examples/UIExplorer
xctool -scheme UIExplorer -sdk iphonesimulator8.1
```

```bash
./scripts/e2e-test.sh
```

```bash
./runXcodeTests.sh
```

But I can't reproduce the [failing test](https://travis-ci.org/facebook/react-native/jobs/63267954#L2138)

I have to retrieve the new screenshot and put it on the **xamples/UIExplorer/UIExplorerTests/ReferenceImages/Examples-UIExplorer-UIExplorerApp/** folder, but I can't launch the test.

Anyone knows ?

**Comment by ericvicenti** [comment]
Ah, so the snapshot tests can be finicky. Things will look different based on the device in the simulator, 32 vs 64 bit, OS version, etc. Are you re-recording the snapshot on iPhone 5 (32-bit) and iOS 8.x?

You should be able to run the tests locally with `./scripts/objc-test.sh`

**Comment by tsunammis** [comment]
```
Are you re-recording the snapshot on iPhone 5 (32-bit) and iOS 8.x?
```

Yeah, I record the snapshot with iOS 8.1 and iPhone 5 but I can't specify 32-bit.

The difference between my screenshot and the oldest, its the text size, mine is more smaller than the oldest. @ericvicenti Do you know why ?

**Comment by tsunammis** [comment]
@ericvicenti it's ok ? 

I changed the “recordMode” from NO to YES inside the UIExplorerTests, the “reference image” has been updated, and I can’t see my changes on the screenshot, but I see the changes when I run the UIExplorer project on Xcode, do you know why ? And by now, the tests pass but it is bullshit according to me.

What is the next steps to merge this feature ?

**Comment by tsunammis** [comment]
@nicklockwood Do you have something to add ?

**Comment by nicklockwood** [comment]
Running tests on iPhone 5 automatically makes them 32-bit, iPhone 5s and above is 64-bit.

I don't know why you wouldn't be seeing the changes in the reference image. That's very strange.

File: Examples/UIExplorer/TabBarIOSExample.js [hunk 1]
```diff
@@ -89,6 +91,9 @@ var TabBarExample = React.createClass({
 });
 
 var styles = StyleSheet.create({
+  tabBar: {
```
Comment (nicklockwood) [inline Examples/UIExplorer/TabBarIOSExample.js]: Remove this?

File: Examples/UIExplorer/TabBarIOSExample.js [hunk 2]
```diff
@@ -48,7 +48,9 @@ var TabBarExample = React.createClass({
 
   render: function() {
     return (
-      <TabBarIOS>
+      <TabBarIOS
+        tintColor={'black'}
```
Comment (nicklockwood) [inline Examples/UIExplorer/TabBarIOSExample.js]: You can just write  tintColor="black" if value is a string.

**Comment by tsunammis** [comment]
@nicklockwood code updated ;-)

Can you try this branch if you see the tab bar in blue on the UIExplorer project ?

**Comment by tsunammis** [comment]
@a2 @nicklockwood @ericvicenti merging is possible ?

**Comment by johanneslumpe** [comment]
This would be nice to get merged! :)

**Comment by brentvatne** [comment]
Looks good to me

**Comment by tsunammis** [comment]
@brentvatne what is the next step ? This could be merged friday during the time afforded to those things ? -> https://gist.github.com/brentvatne/e1ca1e4beaf0135e3178

**Comment by nicklockwood** [comment]
This was actually merged yesterday. Should be included in the next update. Sorry for the wait!

**Comment by brentvatne** [comment]
:+1: thanks @nicklockwood!

**Comment by brentvatne** [comment]
@tsunammis - looks like that won't be needed :smile: hope you join in on Friday!

**Comment by tsunammis** [comment]
@brentvatne thank you !! I hope I will be available friday, what is the timezone ? I'm in Paris.
@nicklockwood thank you, you are the best !!

**Comment by brentvatne** [comment]
@tsunammis - I'll be around at 10am - 3pm PT :smile:

**Pull request closed without merging**